microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/integration/cli/ConfigureMicroProfileOpenApiExtensionTest.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.eap.qe.microprofile.openapi.integration.cli;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.eap.qe.microprofile.openapi.OpenApiManagementClientHelper;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientHelper;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientRelatedException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigUtils;
+import java.io.IOException;
+
+/**
+ * Test cases for MP OpenAPI feature subsystem configuration
+ */
+@RunWith(Arquillian.class)
+public class ConfigureMicroProfileOpenApiExtensionTest {
+
+    /**
+     * @tpTestDetails Testing CLI operations to configure subsystem (add and removal)
+     * @tpPassCrit Uses {@link ManagementClientHelper} API to perform MP OpenAPI feature subsystem initialization and
+     * finalization.
+     * @tpSince EAP 7.4.0.CD19
+     *
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *                                          {@link OnlineManagementClient} API
+     * @throws IOException
+     * @throws ConfigurationException           Wraps exceptions thrown by the internal operation executed by
+     *                                          {@link ConfigUtils} API
+     */
+    @Test
+    public void testAddAndRemoveExtensionAndSubsystem() throws ManagementClientRelatedException, IOException, ConfigurationException {
+        //  MP OpenAPI up
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone())   {
+            OpenApiManagementClientHelper.enableOpenApi(client);
```

Comment:
I suggest to also verify that the subsystem (its adress) exists. See `org.wildfly.extras.creaper.core.online.operations.Operations`.

Reply:
Done.